wolox-books/src/app/store/book.actions.ts

```diff
@@ -0,0 +1,10 @@
+import { createAction } from '@ngrx/store';
+import { Book } from 'app/models/book';
+
+export const addBook = createAction(
+  '[Book-item Component] Add a book to the shopping cart', (book: Book) => book
```

Comment:
This is more like a function name, not a description. This should be something like `[Book] addBook` where you use the reducer (book) inside the [] and then the action name (`addBook` or `AddBook`)